src/data.rs

```diff
@@ -196,3 +196,9 @@ impl EntryInfo {
         unreachable!()
     }
 }
+
+impl Display for ItemUID {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        write!(f, "{}", self.0)
+    }
+}
```

Comment:
This is a toss-up, but in the rest of the code base we do not define `Display` impls for newtype structs and just format `x.0` if we need it.